libs/api-client-tiger/src/gd-tiger-model/typeGuards.ts

```diff
@@ -26,3 +27,9 @@ export function isResultAttributeHeader(
 ): header is AttributeExecutionResultHeader {
     return (header as AttributeExecutionResultHeader).attributeHeader !== undefined;
 }
+
+export function isVisualizationObjectsItem(
+    visualizationObject: VisualizationObjectsItem,
```

Comment:
This type guard is strange, you can only pass items of the type you are guarding as a parameter, right? Meaning you cannot call it from TypeScript so that this returns false (without some casts). I belive the parameter should have the type `unknown`.

Reply:
fixed